src/gc/handletable.cpp

```diff
@@ -314,7 +314,10 @@ OBJECTHANDLE HndCreateHandle(HHANDLETABLE hTable, uint32_t uType, OBJECTREF obje
     }
 #endif // _DEBUG && !FEATURE_REDHAWK
 
-    VALIDATEOBJECTREF(object);
+    if (object != NULL)
```

Comment:
There is a lot of VALIDATEOBJECTREF / VALIDATEOBJECT calls in our codebase. It seems it would be better to put the check into the macro itself instead.